error: implicit declaration of function 'usleep'

Hello I cannot build it on MAC.

`catfood@Catfoods-MacBook urg_library-1.2.6 % make
cd src/ && /Applications/Xcode.app/Contents/Developer/usr/bin/make
gcc -g -O2  -I../include/c -fPIC   -c -o urg_utils.o urg_utils.c
urg_utils.c:218:5: error: implicit declaration of function 'usleep' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
    usleep(1000 * delay_msec);
    ^
1 error generated.
make[1]: *** [urg_utils.o] Error 1
make: *** [all] Error 2
catfood@Catfoods-MacBook urg_library-1.2.6 % `

--- a/current/src/urg_utils.c
+++ b/current/src/urg_utils.c
@@ -12,6 +12,10 @@
 
 #include "urg_utils.h"
 #include "urg_errno.h"
+#include "urg_detect_os.h"
+#if !defined(URG_WINDOWS_OS)
+#include <unistd.h>
+#endif
 #define _USE_MATH_DEFINES
 #include <math.h>
 

